docs/upgrade-notes/migrate-tea-standalone.md

```diff
@@ -0,0 +1,148 @@
+---
+id: migrate_tea_standalone
+title: Migrate TEA deployment to standalone module
+hide_title: false
+---
+
+## Background
+
+Previous versions of Cumulus included deployment of the Thin Egress App (TEA) by default in the `distribution` module. As a result, this forced Cumulus users who wanted to deploy a new version of TEA to wait on a new release of Cumulus that incorporated that release.
+
+In order to give Cumulus users the flexibility to deploy newer versions of TEA whenever they want, deployment of TEA has been removed from the `distribution` module and **Cumulus users must now add the TEA module to their deployment**. [Guidance on integrating the TEA module to your deployment is provided](deployment/thin-egress-app.md), or you can refer to [Cumulus core example deployment code](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/main.tf).
+
+By default, when upgrading Cumulus and moving from TEA deployed via the `distribution` module to deployed as a separate module, your API gateway for TEA would be destroyed and re-created, which could cause outages for any Cloudfront endpoints pointing at that API gateway.
+
+These instructions outline how to modify your state to preserve your existing Thin Egress App (TEA) API gateway when upgrading Cumulus and moving deployment of TEA to a standalone module.
+
+## Notes about state management
+
+These instructions will involve manipulating your Terraform state via `terraform state mv` commands. These operations are **extremely dangerous**, since a mistake in editing your Terraform state can leave your stack in a corrupted state where deployment may be impossible or may result in unanticipated resource deletion.
+
+To mitigate the risk of having to edit your state, we **recommend temporarily switching your deployment to read from local state**. Switching to local state means that changes to Terraform state are only tracked locally, not on your remote state stored in S3. Thus, if you make a mistake, you can abandon the state changes by switching your deployment to read from its remote state.
+
+### Switch from remote to local state
+
+In your `cumulus-tf` directory:
+
+1. Edit `terraform.tf` to comment out your backend settings
+2. Re-run `terraform init` and answer "yes" when asked to copy backend configuration
+
+![Screenshot of terminal showing output from "terraform init" to switch from remote state to local state](assets/switch-to-local-state.png)
```

Comment:
💯  Nice work. 

In fact, I may go upgrade my comments in the disaster recovery PR to be this explicit.

Reply:
When I view the file on Github, these screenshot links are broken:
![image](https://user-images.githubusercontent.com/14986053/93383488-531d5400-f829-11ea-9a0e-28f92f916b51.png)

Reply:
This is probably a larger problem with our docs though, may be out of scope.

Reply:
Yeah, the native markdown parser understand the syntax for referring to image assets that our docs expect. See https://github.com/nasa/cumulus/blob/master/docs/workflows/workflow-triggers.md